packages/injected/src/wallets.ts

```diff
@@ -512,6 +512,19 @@ const gamestop: InjectedWalletModule = {
   platforms: ['desktop']
 }
 
+const core: InjectedWalletModule = {
+  label: ProviderLabel.Core,
+  injectedNamespace: InjectedNameSpace.Ethereum,
+  checkProviderIdentity: ({ provider }) =>
+    !!provider && !!provider[ProviderIdentityFlag.Core],
+  getIcon: async () => (await import('./icons/core.js')).default,
+  getInterface: async () => ({
+    provider: createEIP1193Provider(window.avalanche)
```

Comment:
I faced the same problem it happens one time initially. But I think it will be fixed since they are still developing. But I'll ask devs about this and other issues I faced in terms of UX. Hope to get it fixed soon.